Joystream/pioneer
# 🕴️ Prevent voting with a blacklisted account

- #3874

`@joystream/types` still needs to upgraded or patched

### QA
1. Pick an account to blacklist :warning: this will be permanent on the network it's done on.
2. Make sure the account has at least a bit of fund to pay for transaction fees.
3. In https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fatlas-next.joystream.org%2Fws-rpc#/extrinsics select the account to blacklist and in the "submit the following extrinsic" field: `referendum` ⇒ `optOutOfVoting()`

   ![image](https://user-images.githubusercontent.com/6571453/218091771-90fafd63-d004-4b5b-b7f3-784a51ed5836.png)
   Then "Submit transaction"
4. Finally during the voting stage go to: https://dao-git-feature-blacklisted-accounts-joystream.vercel.app/#/election?network-config=https://atlas-next.joystream.org/network/config.json and check the available accounts in the voting modal.
